src/test/java/org/jabref/logic/openoffice/oocsltext/CSLFormatUtilsTest.java

```diff
@@ -502,10 +499,10 @@ static Stream<Arguments> ooHTMLTransformFromCitationWithMultipleEntries() {
      * Test for modifying the number (index) of a numeric citation.
      * The numeric index should change to the provided "current number".
      * The rest of the citation should stay as it is (other numbers in the body shouldn't be affected).
-     * <p>
-     * <b>Precondition 1:</b> This test assumes that {@link CitationStyleGenerator#generateBibliography(List, String, CitationStyleOutputFormat, BibDatabaseContext, BibEntryTypesManager) generateBibliography} works as expected.<br>
-     * <b>Precondition 2:</b> This test assumes that the method {@link CSLFormatUtils#transformHTML(String) transformHTML} works as expected.<br>
-     * <b>Precondition 3:</b> Run this test ONLY on numeric Citation Styles.</p>
+     *
+     * @implSpec 1. Assumes that {@link CitationStyleGenerator#generateBibliography(List, String, CitationStyleOutputFormat, BibDatabaseContext, BibEntryTypesManager) generateBibliography} works as expected.<br>
+     *           2. Assumes that the method {@link CSLFormatUtils#transformHTML(String) transformHTML} works as expected.<br>
+     *           3. Run this test ONLY on numeric Citation Styles.</p>
```

Comment:
Maybe convert to Markdown (`///` instead of `*`) to have that rendered proply (Otherwise, you should use `<ol>` and `<li>`.